site/contents/meetings/next-meeting.md

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; url=2021/07-july/14-07-2021_meeting.html" />
```

Comment:
This file will need updating each time (I will try and write something that will do it in a GH action later)